truss-chains/truss_chains/remote_chainlet/utils.py

```diff
@@ -623,13 +638,19 @@ async def send_json(self, data: Any) -> None:
         await self._websocket.send_json(data)
 
     async def iter_text(self) -> AsyncIterator[str]:
-        while True:
-            yield await self.receive_text()
+        return self._websocket.iter_text()
 
     async def iter_bytes(self) -> AsyncIterator[bytes]:
-        while True:
-            yield await self.receive_bytes()
+        return self._websocket.iter_bytes()
```

Comment:
It seems like the key difference between looping on `self.receive_bytes()` and simply calling `self._websocket.iter_bytes()` is that the latter catches and ignores `WebSocketDisconnect`. Do we want to swallow the error like this?

Reply:
I feel like delegating to the underlying impl rather than having an opinion on whether the error should not get swallowed makes sense to me? But no strong preference here